cassovary-core/src/main/scala/com/twitter/cassovary/graph/SemiSynchronizedDynamicGraph.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2014 Twitter, Inc.
```

Comment:
Should we not copy the copyright? Or add our own copyright as well to files that we are adding?